src/extension.ts

```diff
@@ -85,8 +85,12 @@ export class Extension implements vscode.Disposable {
       editor.options.lineNumbers = vscode.TextEditorLineNumbersStyle.On
```

Comment:
We still change the style here, which is probably not expected.

Reply:
Is this a problem? I would assume that setting it to `On` when its already `On` is idempotent.

Reply:
It might not be `On` though, for instance because of another extension. To me, the new option means "Dance will not touch line numbers," so we should avoid changing the line numbers if the option is set.

Reply:
I think splitting the options in 2 and adding an 'inherit' option should make this more clear.